dev/tasks/tasks.yml

```diff
@@ -433,7 +433,7 @@ tasks:
 
   # Don't use this for now. We can't complete this task in 50min on Travis CI.
   debian-stretch-arm64:
-    ci: travis
+    ci: azure
     platform: linux
     template: linux-packages/travis.linux.arm64.yml
```

Comment:
Can we enable this too?
We can't run this on Travis CI because it takes 50min+.

Reply:
I'm trying this.